### ISSUE TITLE
[EN][WotLK Raid][Obsidian Sanctum 3 Drakes]

### ISSUE THREAD

**Issue by lamaze** [Atlantiss/BugTracker#8676]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Boss keeps resetting and can't be killed.

**Current behaviour**: When fighting Sartharion on 25man mode with 3 drakes alive (didn't test other modes), the boss randomly resets. (If you killed some drakes meanwhile they reborn and fly back in their start position). We tried more times, and in one of our tries the boss resetted after dying.

**Expected behaviour**: The boss dont have to randomly reset during the fight.

**Steps to reproduce the problem**:

1. Engage Sartharion with 3 drakes alive
2. Fight
3. Wait for the bug to happen ( it will, don't worry)

**Revision**: 5267

**Comment by coba999** [comment]
Someone of your raid moving boss away from fighting zone.

**Comment by DuGalle** [comment]
Yes i agree if boss is moved encounter bugs never let dragon touch lava
